### ISSUE TITLE
html prop doesn't update on rerender

### ISSUE THREAD

**Issue by chasedeanda** [tvkhoa/react-tippy#10]
I am using the `html` prop to pass in some custom html. The html content can change on each re-render, but it looks like once it gets set it won't change. Is there some kind of `shouldComponentUpdate` going on in the component preventing it from updating?

**Comment by chasedeanda** [comment]
Hmm, looks like it was some kind of `key` issue. In order to get the `html` to update I had to pass in a different key.

**Comment by tvkhoa** [comment]
Hi @chasedeanda , could you please provide more info? Or at least, an example so that we can investigate to know why it happened.

**Comment by maxsalven** [comment]
@tvkhoa I think I may have run into the same issue. Here's a basic drop down menu system using react-tippy:

```javascript
import React from "react";
import { Tooltip } from "react-tippy";
import "react-tippy/dist/tippy.css";

const ManualPopover = ({ children, open, onRequestClose, content }) => {
  return (
    <Tooltip
      useContext
      html={content}
      position={"bottom"}
      open={open}
      onRequestClose={onRequestClose}
      trigger="manual"
      interactive
      animateFill={false}
    >
      {children}
    </Tooltip>
  );
};

class Menu extends React.Component {
  state = { visible: false };
  render() {
    const { selected, children } = this.props;

    const PopoverContent = (
      <div onClick={() => this.setState({ visible: false })}>
        {children}
      </div>
    );

    return (
      <ManualPopover
        open={this.state.visible}
        content={PopoverContent}
        onRequestClose={() => {
          this.setState({ visible: false });
        }}
      >
        <button onClick={e => this.setState({ visible: !this.state.visible })}>
          {selected}
        </button>
      </ManualPopover>
    );
  }
}

class Example extends React.Component {
  state = { value: 1 };

  render() {
    const { value } = this.state;
    const options = [1, 2];

    return (
      <Menu selected={value}>
        {options.map(option => {
          const isActive = option === value;
          console.log(`Rendering ${option}, isActive: ${isActive}`);
          return (
            <div key={option} onClick={() => this.setState({ value: option })}>
              {option} – {isActive ? "Active" : "Not Active"}
            </div>
          );
        })}
      </Menu>
    );
  }
}
```

If you 'open' the menu, and select item 2, then you'll see in the console that `isActive` is true for item 2:
<img width="285" alt="screen shot 2017-07-03 at 2 43 38 pm" src="https://user-images.githubusercontent.com/4144910/27780938-5502aff2-5ffe-11e7-9738-099149c84d0c.png">


However, the content of the tooltip has not re-rendered with these new props:
<img width="145" alt="screen shot 2017-07-03 at 2 43 57 pm" src="https://user-images.githubusercontent.com/4144910/27780908-3012df5a-5ffe-11e7-9edb-31d04847fdcd.png">

**Comment by Utzel-Butzel** [comment]
I think @chasedeanda is using a workaround like this: 

`<Tooltip
key={this.state.meta.source}
html={(
<div>{this.state.meta.source}</div>
)}>
Info</Tooltip>`

Whenever `key` is updated `html` is also getting an refresh.

**Comment by tvkhoa** [comment]
@chasedeanda , @Utzel-Butzel 

Fixed in 1.0.2. Thank you.
Feel free to fire an issue if you have any problem with it
